scipy/sparse/array.py

```diff
@@ -0,0 +1,105 @@
+"""
+Sparse array creation functions e.g. `eye`, `diags`, etc.
+"""
+
+from . import _construct
```

Comment:
Should this be:
```
from scipy.sparse import _construct
```

Reply:
Why, is that a pattern that is used elsewhere?

Reply:
We have a mix of relative and absolute imports, but within a module I'd say relative is preferred.